src/foam/u2/borders/CardBorder.js

```diff
@@ -13,8 +13,10 @@ foam.CLASS({
     ^ {
       background-color: #ffffff;
       border: solid 1px #e7eaec;
-      box-shadow: 0 4px 5px 0 rgba(0,0,0,0.14), 0 1px 10px 0 rgba(0,0,0,0.12), 0 2px 4px -1px rgba(0,0,0,0.20);
-      padding: 6px;
+      border-radius: 6px;
```

Comment:
@blakegreen32 Please create a ticket to capture all these colours that need to be supported for Cards - I'm assuming.  Or update them if we have what we need.